contracts/Genesis.sol

```diff
@@ -196,31 +205,25 @@ contract Genesis is ERC721, VRFConsumerBase, Ownable {
     }
 
     /**
-     * Enumerable
-     */
-    // TODO Might want to add this one at some point. We would probably need a method to know the token types too.
-    // function tokensOfOwner(address _owner)
-    //     external
-    //     view
-    //     returns (uint256[] memory)
-    // {
-    //     uint256 tokenCount = balanceOf(_owner);
-    //     uint256[] memory tokensId = new uint256[](tokenCount);
-    //     for (uint256 i = 0; i < tokenCount; i++) {
-    //         tokensId[i] = tokenOfOwnerByIndex(_owner, i);
-    //     }
-
-    //     return tokensId;
-    // }
-
-    /**
-     * Withdrawing
+     * Withdraw balance from the contract
      */
-    // TODO Add the logic to distribute funds amongst the team
-    function withdraw() public payable onlyOwner {
+    function withdrawAll() public payable onlyOwner {
         uint256 balance = address(this).balance;
         require(balance > 0, "No ether left to withdraw");
         (bool success, ) = (msg.sender).call{value: balance}("");
         require(success, "Transfer failed.");
     }
+
+    /**
+     * Modifier that limits the function to WL members only
+     * @param nonce nonce to be used
+     * @param proof proof
+     * Owner bypasses this (for tests)
+     */
+    modifier onlyWhitelist(uint256 nonce, bytes32[] memory proof) {
```

Comment:
Well, to be fair the owner is gonna be a gnosis safe, which can't mint. Also we control the owner, so if WE mint with this during presale, we are retarded.
Nevertheless, I agree owner does not need to bypass this. It was mostly for tests. If we find how to set address for local nodes, I can remove this.

Reply:
Add a comment in that case.
We should not write the contract thinking how it will be used in the wild, we have some control over that, but I think the security of the contract should be built in and not assuming how it will be used.

> If WE mint with this during presale, we are retarded.
I agree, but why not make it more secure directly?